tools/src/versioning/android/versionVendoredModules.ts

```diff
@@ -219,14 +199,17 @@ async function baseTransformsFactoryAsync(prefix: string): Promise<Required<File
       },
       {
         paths: 'build.gradle',
-        find: /\b(compileOnly|implementation)\s+['"]com.facebook.react:react-native:.+['"]/gm,
+        find: /\b(compileOnly|implementation|api)\s+['"]com.facebook.react:react-native:.+['"]/gm,
         replaceWith:
           `implementation 'host.exp:reactandroid-${prefix}:1.0.0'` +
           '\n' +
           // Adding some compile time common dependencies where the versioned react-native AAR doesn't expose
-          `    compileOnly 'com.facebook.fbjni:fbjni:${fbjniVersion}'\n` +
-          `    compileOnly 'com.facebook.yoga:proguard-annotations:${proguardAnnotationVersion}'\n` +
-          `    compileOnly 'androidx.annotation:annotation:+'\n`,
+          `    compileOnly 'com.facebook.fbjni:fbjni:+'\n` +
+          `    compileOnly 'com.facebook.yoga:proguard-annotations:+'\n` +
+          `    compileOnly 'com.facebook.soloader:soloader:+'\n` +
+          `    compileOnly 'androidx.annotation:annotation:+'\n` +
+          `    compileOnly 'com.google.code.findbugs:jsr305:+'\n` +
+          `    compileOnly 'androidx.appcompat:appcompat:+'\n`,
```

Comment:
instead of reading the versions from expoview and paying efforts to maintain the versions. we could use `+` to specify any versions.